python/src/aac/plugins/validators/root_keys/_validate_root_keys.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+from aac.lang.language_context import LanguageContext
+from aac.lang.active_context_lifecycle_manager import get_active_context
+from aac.lang.definitions.definition import Definition
+from aac.plugins.validators import ValidatorResult
+
+
+def validate_root_keys(definition_under_test: Definition, target_schema_definition: Definition, active_context: LanguageContext, *validation_args) -> ValidatorResult:
+    """
+    Validates that the definition root key is defined by the root definition.
+
+    Args:
+        definition_under_test (Definition): The definition that's being validated.
+        target_schema_definition (Definition): A definition with applicable validation.
+        active_context (LanguageContext): The active context.
+
+    Returns:
+        A ValidatorResult containing any applicable error messages.
+    """
+    error_messages = []
+    active_context_keys = get_active_context().get_root_keys()
+
+    # We're expecting only full definition dicts -- so the root key is the first value
```

Comment:
We should encode this expectation in a test rather than a comment.

Reply:
This comment is old. 🙃
```suggestion
```